.gitlab-ci.yml

```diff
@@ -126,6 +126,7 @@ typecheck:
   interruptible: true
   script:
     - yarn
+    - yarn build
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏since we now always build before typecheck, what about typechecking the test app and e2e tests in this job as well?